client/app/dedicatedCloud/datacenter/backup/enable/dedicatedCloud-datacenter-backup-enable.controller.js

```diff
@@ -26,19 +26,26 @@ angular
       this.datacenter = null;
       this.hosts = null;
 
+      return this.fetchInitialData();
+    }
+
+    fetchInitialData() {
       this.loading.init = true;
 
-      return this.$q.all({
-        datacenter: this.DedicatedCloud
-          .getDatacenterInfoProxy(this.$stateParams.productId, this.$stateParams.datacenterId),
-        hosts: this.DedicatedCloud
-          .getHostsLexi(this.$stateParams.productId, this.$stateParams.datacenterId),
-      }).then((data) => {
-        this.datacenter = data.datacenter;
-        this.hosts = data.hosts;
-      }).finally(() => {
-        this.loading.init = false;
-      });
+      return this.$q
+        .all({
+          datacenter: this.DedicatedCloud
+            .getDatacenterInfoProxy(this.$stateParams.productId, this.$stateParams.datacenterId),
+          hosts: this.DedicatedCloud
+            .getHostsLexi(this.$stateParams.productId, this.$stateParams.datacenterId),
```

Comment:
I know you've not written this part but maybe `getHostsLexi` should be renamed as it refers to legacy naming

Reply:
I agree with destroying all mentions to `Lexi`

Reply:
It's been done